### DISCUSSION TITLE
DigiByte Core v8.22.0-RC4

### DISCUSSION THREAD

**Discussion by JaredTate** [DigiByte-Core/digibyte#242] [General]
Here is a place for everyone to discuss all things related to RC4. Please report bugs, and issues and provide any feedback you wish. Thank you all for helping to test RC4.

**Comment by wpbloger** [comment]
Hello. I wanted to test this new version, but when I command ./digibyte-cli getblocktemplate I get this error code: -8 error message: getblocktemplate must be called with the segwit rule set (call with {"rules": ["segwit"] }), but the command ./digibyte-cli getnetworkinfo works correctly. 
```
{
  "version": 82200,
  "subversion": "/DigiByte:8.22.0/",
  "protocolversion": 70018,
  "localservices": "0000000000000409",
  "localservicesnames": [
    "NETWORK",
    "WITNESS",
    "NETWORK_LIMITED"
  ],
  "localrelay": true,
  "timeoffset": 0,
  "networkactive": true,
  "connections": 10,
  "connections_in": 0,
  "connections_out": 10,
  "networks": [
    {
      "name": "ipv4",
      "limited": false,
      "reachable": true,
      "proxy": "",
      "proxy_randomize_credentials": false
    },
    {
      "name": "ipv6",
      "limited": false,
      "reachable": true,
      "proxy": "",
      "proxy_randomize_credentials": false
    },
    {
      "name": "onion",
      "limited": true,
      "reachable": false,
      "proxy": "",
      "proxy_randomize_credentials": false
    },
    {
      "name": "i2p",
      "limited": true,
      "reachable": false,
      "proxy": "",
      "proxy_randomize_credentials": false
    }
  ],
  "relayfee": 0.00100000,
  "incrementalfee": 0.00010000,
  "localaddresses": [
  ],
  "warnings": ""
}
```
But even here I don’t like the fact that “connections_in”: 0. Which port do I need to open so that my local node accepts incoming connections?
How can I fix this?
And I also want to know why there are so few “connections_out”: 10,

Here is my config file:
```
algo=sha256d
rpcuser=user
rpcpassword=mupass
rpcport=17024
port=12021
server=1
txindex=1
debug=1
maxconnections=128
daemon=0
```

**Comment by chopperbriano** [comment]
What happens if you run the command like this?

getblocktemplate "{\"rules\":[\"segwit\"]}"
